### PR TITLE
feature(#3) - Set up Redux Toolkit 

### PR DESCRIPTION
**Pull request by avilalte**
**Changes**
- Installed dependencies for Redux Toolkit.
- Added a form on the user page that receives user data and dispatches an action to the corresponding store reducer upon submission.

**To test this:**
- Open a terminal and navigate to the project root.
- Run `yarn run dev`
- Go to the users page. Fill out the form and submit it. The new user will appear at the top of the list below.

**Checklist**
- [x] Commit messages follow the standars
- [x] I linked the related issue
- [x] I added reviewers
- [x] PR name follows the standards
- [x] I reviewed my PR

### CODE REVIEW

**Review by Qweffy — COMMENTED** [review]
Take a look at the comments

File: src/components/DumbComponent.tsx [hunk 1]
```diff
@@ -1,26 +1,22 @@
...
-}
 
 // Props interface with required users property
 interface Props {
-  users: User[];
+  users: User[]
 }
 
 // Function component UserCards with props of type Props
 const DumbComponent: React.FC<Props> = (props) => {
```
Comment (Qweffy) [inline src/components/DumbComponent.tsx:12]: No need to use React.FC type if you{re actually sending the Props at the copmpnent, TS it's going to infer it is a jsx component

File: src/components/DumbComponent.tsx [hunk 2]
```diff
@@ -1,26 +1,22 @@
...
 
 // User interface with required properties
-interface User {
-  name: string;
-  age: number;
-  genre: string;
-}
 
 // Props interface with required users property
 interface Props {
```
Comment (Qweffy) [inline src/components/DumbComponent.tsx:7]: The name of the Props should be DumbComponentProps; it's going to be easier identify it or look for it later

**Comment by avilalte** [comment]
- I updated the DumbComponent so it has the users form that was previously rendered on the users page. 
- Now the DumbComponent is rendered in the test-page.

**Pull request merged**